ogticrd/covid-safe-paths
# changin datePicker design on android

#### Description:


I have changed the design of datePicker to another friendly one, I have used the one Facebook has used

#### Linked issues:


Fix: [140](https://trello.com/c/9OIiEeyQ/140-revisar-formato-de-fecha-del-form-para-que-sea-m%C3%A1s-end-user-friendly)

#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/84524839-96a37780-aca8-11ea-81f5-f1b1785f6fb2.png)


#### How to test:


- `git pull && git checkout Fix/140-friendly-date-picker`
- Head over to report screen and press one of the option to start to fill it
- Check the date picker design
